docs/proposals/configmap.md

```diff
@@ -293,12 +296,12 @@ spec:
         - name: config-map-volume
           mountPath: /mnt/config-map
   volumes:
-  - name: config-map-volume
-    configMap:
-      files:
-        - path: "etc/redis.conf"
-          configMapName: redis-volume-config
-          key: redis.conf
+    - name: config-map-volume
+      configMap:
+        configMapName: redis-volume-config
```

Comment:
...and if we do, Secret should change, too.  We can always change later, I guess, by marking this field as optional (which Secret does with SecretName) and later changing to one-of SecretName or [Namespace]+Name